Doc/library/pathlib.rst

```diff
@@ -933,23 +933,32 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.rename(target)
 
-   Rename this file or directory to the given *target*.  On Unix, if
-   *target* exists and is a file, it will be replaced silently if the user
-   has permission.  *target* can be either a string or another path object::
+   Rename this file or directory to the given *target*, and return a new Path
+   instance pointing to *target*.  On Unix, if *target* exists and is a file,
+   it will be replaced silently if the user has permission.  *target* can be
+   either a string or another path object::
 
       >>> p = Path('foo')
       >>> p.open('w').write('some text')
       9
       >>> target = Path('bar')
       >>> p.rename(target)
+      PosixPath('bar')
```

Comment:
Hmm. I wonder if this docstring will fail doctests on Windows (or violate Windows users' expectations). @zooba can you comment?

Reply:
On deeper inspection, I see that `PosixPath` is what the docs above use, so this string is the most consistent with the current behavior. If these docs are tested, they're not tested on Windows.